flax/nnx/graph.py

```diff
@@ -258,6 +285,11 @@ def __treescope_repr__(self, path, subtree_renderer):
       subtree_renderer=subtree_renderer,
     )
 
+if config.flax_use_flaxlib:
```

Comment:
Why does it need to import repeatedly? Can we do one import at top?

Reply:
could be done once at the end I guess, else they are rewritten by the actual definition